src/Writers/HTMLWriter.jl

```diff
@@ -1405,6 +1405,30 @@ function generate_version_file(versionfile::AbstractString, entries, symlinks =
     end
 end
 
+# write redirect file
+function generate_redirect_file(redirectfile::AbstractString, entries, devurl, versions)
+    # If `versions` contains "stable"=>"v^" and `entries` contains "stable", then version="stable".
+    # In other cases, version="dev".
+
+    comment = "<!--This file is automatically generated by Documenter.jl-->"
+
+    isfile(redirectfile) && !startswith(read(redirectfile, String), comment) && return
+
+    version = devurl
+    stable_index = findfirst(v -> last(v)=="v^", versions)
+    if stable_index !== nothing
+        stable_version = first(versions[stable_index])
+        if stable_version in entries
+            version = stable_version
+        end
+    end
```

Comment:
Can this reuse some functinality for the "outdated warning" code? Makes sense that those would point to the same version.

Reply:
Are you suggesting to reuse the script of `generate_version_file`

https://github.com/JuliaDocs/Documenter.jl/blob/v0.27.4/src/Writers/HTMLWriter.jl#L1396-L1403

or `warner.js`?

https://github.com/JuliaDocs/Documenter.jl/blob/v0.27.4/assets/html/warner.js

I think the redirect page should be `./stable` or `./dev`, so we can't reuse the script here.
Sorry if I misunderstood something.

Reply:
I meant that `index.html` should redirect to the same thingthe outdated warning rediretion does.

Reply:
Ah, I see. All I had to do was just picking the first element of `entries`.

https://github.com/JuliaDocs/Documenter.jl/blob/v0.27.4/src/Writers/HTMLWriter.jl#L1404
https://github.com/JuliaDocs/Documenter.jl/blob/v0.27.4/assets/html/warner.js#L40